src/Microsoft.Identity.Client/Internal/Requests/AuthenticationRequestParameters.cs

```diff
@@ -62,6 +62,7 @@ internal class AuthenticationRequestParameters
 
             // Set application wide query parameters.
             ExtraQueryParameters = serviceBundle.Config.ExtraQueryParameters ?? new Dictionary<string, string>();
+            Claims = serviceBundle.Config.Claims;
```

Comment:
@MarkZuber - I don't fully undersand the flow here. I was expecting the commonParameters.Claims to be populated, for example when doing the U/P setup. It is not.